src/main/java/biz/paluch/logging/gelf/intern/sender/QueryStringParser.java

```diff
@@ -119,4 +119,47 @@ public static String getHost(URI uri) {
         }
         return host;
     }
+
+    public static Properties parseKafkaHost(String host) {
+        Properties properties = new Properties();
```

Comment:
We should convert the `host` value into an `URI` to get some assistance from existing implementations. See [here](https://github.com/lettuce-io/lettuce-core/blob/master/src/main/java/io/lettuce/core/RedisURI.java#L903) for an example how to parse comma-separated host values.